### DISCUSSION TITLE
mspider/troubleshoot/sidecar

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2926] [Announcements]
# mspider/troubleshoot/sidecar

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/troubleshoot/sidecar.html

**Comment by hanxiaop** [comment]
考虑添加 Webhook 无法工作的问题排查，具体表现为 Internal error occurred failed callling webhook，也经常会遇到。这种情况不是 Istio 的问题，需要用户排查网络/防火墙。

**Comment by hanxiaop** [comment]
可以考虑添加建议使用 `istioctl analyze` 排查，可以检查一些服务/策略配置的问题。
